### ISSUE TITLE
TypeError: Invalid param, Expected String

### ISSUE THREAD

**Issue by jacquesdev** [Instabug/Instabug-React-Native#362]
### Steps to Reproduce the Problem
Pass in an empty string
Instabug.setUserAttribute('anykey', '')

### Expected Behavior
I would expect it to either accept an empty string or to have a better error because it should be a value error, in this case, assuming Instabug does not want to accept an empty string.

### Actual Behavior
TypeError: Invalid param, Expected String

### Instabug integration code
 ```
Instabug.startWithToken(INSTABUG_KEY, [Instabug.invocationEvent.none])
Instabug.identifyUser(email, name.first_name)
Instabug.setUserAttribute('country', country)
```

### SDK Version
8.5.1

### React Native, iOS and Android Versions
React Native Environment Info:
    System:
      OS: macOS 10.14.6
      CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
      Memory: 20.14 MB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
      Yarn: 1.15.2 - ~/.yarn/bin/yarn
      npm: 6.10.2 - ~/.nvm/versions/node/v10.15.3/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
      Android SDK:
        API Levels: 28, 29
        Build Tools: 28.0.3, 29.0.2
        System Images: android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5692245
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.9 => 0.59.9 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7

**Comment by HeshamMegid** [comment]
Thanks for reporting this. We're looking into it and will get back to you soon.

**Comment by AliAbdelfattah** [comment]
The issue has been fixed in v8.5.6.
